source/NVDAObjects/window/scintilla.py

```diff
@@ -193,9 +194,9 @@ def _getLineOffsets(self,offset):
 			# Lines in Scintilla refer to document lines, not wrapped lines.
 			# There's no way to retrieve wrapped lines, so use screen coordinates.
 			y=watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,offset)

```

Comment:
You now can retrieve y with _getPointFromOffset. That gives you an x you won't use, but it at least ensures that you're following an uniform codepath to get screen coordinates from scintilla offsets.